jmix-reports/reports-flowui/src/main/java/io/jmix/reportsflowui/view/run/ReportTableView.java

```diff
@@ -213,6 +229,10 @@ protected String findTableCode(Report report) {
         return null;
     }
 
+    /**
+     * @deprecated use {@link #drawTables(ReportOutputDocument)}
+     */
+    @Deprecated(since = "2.1.3", forRemoval = true)
```

Comment:
There is a proposal to move this fix to a major release. The result is a lot of deprecated code that can break overridden methods for users (for example `drawTable`).

Reply:
Agree